aws-iot-securityprofile/src/test/java/com/amazonaws/iot/securityprofile/TranslatorTest.java

```diff
@@ -1,20 +1,19 @@
 package com.amazonaws.iot.securityprofile;
 
+import com.google.common.collect.ImmutableSet;
+import org.junit.jupiter.api.Test;
+import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
+
+import java.util.ArrayList;
+import java.util.Set;
+
 import static com.amazonaws.iot.securityprofile.TestConstants.BEHAVIOR_NAME;
 import static com.amazonaws.iot.securityprofile.Translator.translateBehaviorListFromIotToCfn;
 import static com.amazonaws.iot.securityprofile.Translator.translateBehaviorSetFromCfnToIot;
 import static com.amazonaws.iot.securityprofile.Translator.translateMetricValueFromCfnToIot;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-import java.util.ArrayList;
-import java.util.Set;
-
-import com.google.common.collect.ImmutableSet;
-
-import org.junit.jupiter.api.Test;
-import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
-
 public class TranslatorTest {
 
     @Test
```

Comment:
Maybe add test for the custom metric changes made to the Translator for supporting new metric types.

Reply:
+1

Reply:
Added translator verification tests